src/components/PaymentLog.vue

```diff
@@ -1025,7 +1025,7 @@ export default {
       } else this.addDownpayment = false
       this.stillShowToggle = this.addDownpayment
 
-      if(this.selectedProduct.price < 100000 && this.productPlans.includes(this.salesLogForm?.business_type_id?.slug)){
+      if(this.selectedProduct.price < 100000 && this.productPlans.includes(this.salesLogForm?.business_type_id?.slug) && process.env.VUE_APP_SANDBOX === 'true'){
```

Comment:
i think if we just do `process.env.VUE_APP_SANDBOX`, that should be fine

Reply:
na. it sees it as a string, i tried it with yesterdays work